.github/workflows/publish-binary.yml

```diff
@@ -64,58 +63,32 @@ jobs:
           mv ../build/moonbeam-x86-64 ../build/moonbeam 
           yarn
           yarn -s run ts-node github/generate-release-body.ts --owner "${{ github.repository_owner }}" --repo "$(basename ${{ github.repository }})" --from "${{ github.event.inputs.from }}" --to "${{ github.event.inputs.to }}" --srtool-report-folder '../build/' > ../body.md
-      - name: Create draft release
+      - name: Prepare binary assets
+        run: |
+          # Prepare binaries for upload
+          # x86-64 binary gets renamed to just "moonbeam"
+          cp build/moonbeam-x86-64 moonbeam
+          # Other architectures keep their suffix
+          cp build/moonbeam-skylake moonbeam-skylake
+          cp build/moonbeam-znver3 moonbeam-znver3
+      - name: Create draft release with binaries
         id: create-release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v2
         with:
           tag_name: ${{ github.event.inputs.to }}
-          release_name: Moonbeam ${{ github.event.inputs.to }}
+          name: Moonbeam ${{ github.event.inputs.to }}
           body_path: body.md
           draft: true
-
-  ####### Upload Binaries #######
-
-  upload-binaries:
-    runs-on: ubuntu-latest
-    needs: ["build-binary", "publish-draft-release"]
-    strategy:
-      matrix:
-        cpu: ["x86-64", "skylake", "znver3"]
```

Comment:
Lets try to keep this `cpu: ["x86-64", "skylake", "znver3"]`

Reply:
I experimented a bit with this, but didn't find an elegant way to use this var. Mostly because of the special treatment for the `x86-64` target.